Description: Don't use capture when running Core tests on 32-bit architectures.
 Otherwise, we run out of memory.
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1834g
Last-Update: 2021-01-18

--- a/M2/Macaulay2/m2/examples.m2
+++ b/M2/Macaulay2/m2/examples.m2
@@ -116,6 +116,7 @@
     -- TODO: remove this when the effects of capture on other packages is reviewed
     (isTest or match({"FirstPackage", "Macaulay2Doc"},            pkg#"pkgname"))
     and not match({"EngineTests", "ThreadedGB", "RunExternalM2", "DiffAlg"}, pkg#"pkgname")
+    and not (pkg#"pkgname" == "Core" and version#"pointer size" == 4)
     -- FIXME: these are workarounds to prevent bugs, in order of priority for being fixed:
     and not match("(gbTrace|NAGtrace)",                       inputs) -- cerr/cout directly from engine isn't captured
     and not match("(notify|stopIfError|debuggingMode)",       inputs) -- stopIfError and debuggingMode may be fixable
